plugins/woocommerce/client/admin/client/marketplace/components/my-subscriptions/types.ts

```diff
@@ -33,6 +33,7 @@ export type Subscription = {
 	is_installable: boolean;
 	is_shared: boolean;
 	owner_email: boolean;
+	has_changelog: boolean;
```

Comment:
Might there be an edge case when `subscriptions` will be cached with the old schema? Should only by temporary anyway so not sure if we should spend the extra time to address it